test/test_binary_ufuncs.py

```diff
@@ -1687,13 +1687,11 @@ def test_cpu_tensor_pow_cuda_scalar_tensor(self, device):
             self._test_pow(base, exp)
 
     @onlyCUDA
-    @dtypes(torch.complex64, torch.complex128)
-    def test_pow_cuda_complex_extremal_failing(self, device, dtype):
-        t = torch.tensor(complex(-1.0, float("inf")), dtype=dtype, device=device)
-        with self.assertRaises(AssertionError):
-            cuda_out = t.pow(2)
-            cpu_out = t.cpu().pow(2)
-            self.assertEqual(cpu_out, cuda_out)
+    def test_pow_cuda_complex_extremal_passing(self, device):
```

Comment:
Would it make sense to preserve the original case instead of changing the inputs here?